litho-it/src/test/java/com/facebook/litho/widget/RecyclerEventsControllerTest.java

```diff
@@ -72,52 +86,61 @@ public void testOnRecyclerListener() {
 
   @Test
   public void testClearRefreshingOnNotRefreshingView() {
-    when(mSectionsRecyclerView.isRefreshing()).thenReturn(false);
-
+    setUpInitialValues(false);
     mRecyclerEventsController.clearRefreshing();
 
-    verify(mSectionsRecyclerView, never()).setRefreshing(anyBoolean());
-    verify(mSectionsRecyclerView, never()).removeCallbacks(any(Runnable.class));
-    verify(mSectionsRecyclerView, never()).post(any(Runnable.class));
+    assertThat(mSectionsRecyclerView.getSetRefreshingValues()).isEmpty();
+    verifyRemoveCallbacksWasCalledNTimes(mSectionsRecyclerView, 0);
+    verifyPostWasCalledNTimes(mSectionsRecyclerView, 0);
   }
 
   @Test
   public void testClearRefreshingFromUIThread() {
-    when(mSectionsRecyclerView.isRefreshing()).thenReturn(true);
+    setUpInitialValues(true);
     ThreadUtils.setMainThreadOverride(ThreadUtils.OVERRIDE_MAIN_THREAD_TRUE);
-
     mRecyclerEventsController.clearRefreshing();
 
-    verify(mSectionsRecyclerView).setRefreshing(false);
-    verify(mSectionsRecyclerView, never()).removeCallbacks(any(Runnable.class));
-    verify(mSectionsRecyclerView, never()).post(any(Runnable.class));
+    assertThat(mSectionsRecyclerView.getSetRefreshingValues().size()).isEqualTo(1);
+    assertThat(mSectionsRecyclerView.getSetRefreshingValues().get(0)).isFalse();
```

Comment:
And here we can handle it the same as with `refreshListener`, `itemAnimator` and `touchInterceptor`- return last value with getter and just verifying correct result from this getter

Reply:
Done. Verifying the last refresh value